pct.sh

```diff
@@ -26,8 +26,10 @@ exec java \
 	--war "$(pwd)/target/megawar-$LINE.war" \
 	--include-plugins "${PLUGINS}" \
 	--working-dir "$(pwd)/target/pct-work" \
+	--maven-args "-Pconsume-incrementals"
```

Comment:
Doesn't this need a backslash at the end of the line to escape the new line?

Reply:
```suggestion
	--maven-args "-Pconsume-incrementals" \
```